src/sql/workbench/services/objectExplorer/browser/dragAndDropController.ts

```diff
@@ -117,6 +117,14 @@ export class ServerTreeDragAndDrop implements IDragAndDrop {
 		return this.getProviderNameFromElement(element.parent);
 	}
 
+	private getFinalStringForKusto(nodeTypeId: string, escapedName: string) {
+		if (nodeTypeId === 'Table' && escapedName.indexOf(' ') > 0) {
```

Comment:
Why only tables? The doc seems to indicate any object could have spaces - is that incorrect?

Reply:
Great point. Updated!